### PR TITLE
Track lukapona data

### PR DESCRIPTION
**Pull request by gregdan3**

### CODE REVIEW

File: .scripts/fingerspelling_converter.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        data[entry_id] = entry
+
+    # Sort by id, case insensitive
+    data = {k: v for k, v in sorted(data.items(), key=lambda x: x[0].lower())}
+
+    return data
+
+
+def main():
+    os.makedirs("../fingerspelling", exist_ok=True)
```
Comment (TheOnlyTails) [inline .scripts/fingerspelling_converter.py:126]: should we put all the files related to luka pona under a single folder?

possible arrangements:
```
luka_pona
|-- fingerspelling
|   |-- [gloss].toml
|-- signs
|   |-- [gloss].toml
```
etc. localization files would be as they are now

**Comment by TheOnlyTails** [comment]
the workflow is failing because we don't have any localization data yet, should be fixed once the actual files are created.

**Review by TheOnlyTails — COMMENTED** [review]
This is ready, as far as I'm concerned. I've set up Crowdin accordingly (in the `luka_pona` branch there) and all the workflows are working.

**Review by gregdan3 — COMMENTED** [review]
Everything looks good!

**Pull request merged**